src/commands/friends/FriendCommand.ts

```diff
@@ -13,3 +13,83 @@ friend list [@Joker#3650]
 friend requests [incoming/outgoing]
 
 */
+
+import { Command, CommandMessage, CommandoClient } from 'discord.js-commando'
+import { Message, User } from 'discord.js'
+import axios from 'axios'
+
+module.exports = class FriendCommand extends Command {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'friend',
+      group: 'friends',
+      memberName: 'friend',
+      description:
+        'Allows you to send and respond to friend requests, as well as list your friends/friend requests.',
+      details: `
+        \`friend add <mention>\` sends a friend request to that user.\n
+        \`friend accept <mention>\` accepts a friend request from that user.\n
+        \`friend deny/decline <mention>\` denies a friend request from that user.\n
+        \`friend remove/delete <mention>\` removes that user from your friend list.\n
+        \`friend list [mention]\` lists all the user's friends, or your own if no user is given.\n
+        \`friend requests [incoming/outgoing]\` lists all of your incoming or outgoing friend requests, respectfully.
+        If no type is specified, it will list both incoming and outgoing friend requests.
+			`,
+      examples: [
+        'friend add @Joker#3650',
+        'friend deny @Joker#3650',
+        'friend list',
+        'friend requests incoming'
+      ],
+      args: [
+        {
+          key: 'subcommand',
+          label: 'sub-command',
+          prompt:
+            'Would you like to `add/remove/list` friends, `accept/deny` requests, or list `requests`?',
+          type: 'string',
+          infinite: false
+        },
+        {
+          key: 'argument',
+          label: 'user or type',
+          prompt: 'Please provide a valid argument for the used subcommand.',
+          default: -1,
+          type: 'user|string',
+          infinite: false
+        }
+      ]
+    })
+  }
+
+  async run(
+    msg: CommandMessage,
+    { subcommand, argument }: { subcommand: string; argument: User | string }
+  ): Promise<Message> {
+    if (subcommand === 'add') {
+      if (!(argument instanceof User)) {
+        return msg.reply(
+          'This command requires you to specify a user. Please try again.'
+        ) as Promise<Message>
```

Comment:
The type cast can be removed. It's redundant as `msg.reply` already returns a `Promise<Message>`